tasks/console-errors.js

```diff
@@ -74,7 +74,7 @@ const checkPage = async (url) => {
       } else interceptedRequest.continue();
     });
 
-  await page.goto(url, { waitUntil: 'domcontentloaded' });
+  await page.goto(url, { waitUntil: 'networkidle0' });
```

Comment:
Could we add a comment on what `networkidle0` does?

Something like this maybe: https://pptr.dev/api/puppeteer.puppeteerlifecycleevent ?